### PR TITLE
Task7: Implicits

### PR DESCRIPTION
**Pull request by Viachaslau-Zinkevich**

### CODE REVIEW

File: task5/.bsp/sbt.json [hunk 1]
```diff
@@ -1 +1 @@
-{"name":"sbt","version":"1.4.7","bspVersion":"2.0.0-M5","languages":["scala"],"argv":["D:\\Dev\\java\\jdk-11.0.6/bin/java","-Xms100m","-Xmx100m","-classpath","D:/Dev/Tools/JetBrains/Toolbox/apps/IDEA-U/ch-1/203.6682.168.plugins/Scala/launcher/sbt-launch.jar","xsbt.boot.Boot","-bsp"]}
\ No newline at end of file
+{"name":"sbt","version":"1.4.7","bspVersion":"2.0.0-M5","languages":["scala"],"argv":["D:\\Dev\\java\\jdk-11.0.6/bin/java","-Xms100m","-Xmx100m","-classpath","D:/Dev/Tools/JetBrains/Other/config/plugins/Scala/launcher/sbt-launch.jar","xsbt.boot.Boot","-bsp"]}
```
Comment (nikalaikina) [inline task5/.bsp/sbt.json]: Shouldn't it be git ignored?

File: task7/src/main/scala/com/bootcamp/task/Homework.scala [hunk 2]
```diff
@@ -0,0 +1,109 @@
...
+		User("1", "Oleg").show
+	}
+
+	object Task3 {
+		type Error = String
+		trait Parse[T] { // invent any format you want or it can be csv string
+			def parse(entity: String): Either[Error, T]
+		}
+
+		object CsvParseSpec {
```
Comment (nikalaikina) [inline task7/src/main/scala/com/bootcamp/task/Homework.scala]: Why spec?

**Review by nikalaikina — APPROVED** [review]

File: task7/src/main/scala/com/bootcamp/task/Homework.scala (hunk 2 above)
Comment (Viachaslau-Zinkevich) [reply task7/src/main/scala/com/bootcamp/task/Homework.scala]: I was thinking of something else, ment Syntax :)

File: task5/.bsp/sbt.json (hunk 1 above)
Comment (Viachaslau-Zinkevich) [reply task5/.bsp/sbt.json]: It should..

**Pull request merged**